lib/completion.py

```diff
@@ -36,24 +35,13 @@ def _additional_info(self, completion):
         """Provide additional information about the completion object."""
         if completion._definition is None:
             return ''
-        t = completion.type
-        if t == 'statement':
+        if completion.type == 'statement':
             nodes_to_display = ['InstanceElement', 'String', 'Node', 'Lambda',
                                 'Number']
-            desc = ''.join(c.get_code() for c in
+            return ''.join(c.get_code() for c in
                            completion._definition.children if type(c).__name__
                            in nodes_to_display).replace('\n', '')
-        elif t == 'keyword':
-            desc = ''
-        elif t == 'function' and hasattr(completion, 'params'):
-            desc = ', '.join([p.description for p in completion.params])
-        elif t == 'import':
-            desc = ''
-        else:
-            desc = '.'.join(unicode(p) for p in completion._path())
```

Comment:
It looks like right column is often empty, why not to fill it with possibly useful information like source path?